pandas/core/groupby/generic.py

```diff
@@ -1691,17 +1697,17 @@ def _wrap_transformed_output(
 
         return result
 
-    def _wrap_agged_blocks(self, items, blocks):
+    def _wrap_agged_blocks(self, blocks: "List[Block]", items: Index) -> DataFrame:
```

Comment:
I've inverted the argument order. I think it makes more sense to have the blocks come first, like in ``BlockManager``.